examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,111 @@
+raw = mne.io.read_raw_fif(raw_fname, preload=True).pick_types(
+    meg=False, stim=False, eeg=True).filter(1, None, method='iir')
+seed_channel = 'EEG 011'  # Choosing a frontal electrode
+ix_seed = mne.pick_channels(raw.ch_names, [seed_channel])[0]
+ix_targets = np.setdiff1d(range(len(raw.ch_names)), [ix_seed])
+
+# Set up encoding model variables
+X = raw[:][0][ix_targets]
+y = raw[:][0][ix_seed]
+n_targets, n_times = X.shape
+
+###############################################################################
+# Fit the model / Make predictions
+# Define model preprocessing and parameters
+ix_train = np.arange(n_times // 2)
+ix_test = np.arange(n_times // 2, n_times)
+
+# This creates time-lagged versions of the data (in seconds)
+delay_step = .01  # Time between delays (in seconds)
+delayer = FeatureDelayer(sfreq=raw.info['sfreq'],
+                         delays=np.arange(-.1, .1, delay_step))
+info_new = raw.copy().drop_channels([seed_channel]).info.copy()
+info_new['sfreq'] = 1. / delay_step
+
+# This is the final estimator that will be called after masking data
+estimator = make_pipeline(StandardScaler(), Ridge(alpha=1.))
+# This allows us to use a subset of indices for training / prediction
+masker = SampleMasker(estimator, ixs=ix_train, ixs_pred=ix_test)
+# Finally, place a delayer first so that samples are delayed before being split
+pipeline = make_pipeline(delayer, masker)
+
+# Fit / predict with the model
+pipeline.fit(X.T, y)  # estimators expect data of shape (n_samples, n_features)
+y_pred = pipeline.predict(X.T)
+
+###############################################################################
+# Plot raw data
+fig, (ax_raw, ax_pred) = plt.subplots(2, 1, figsize=(8, 8))
+plt_times = raw.times[:200]
+for idata, alpha, lw, color in zip([X, y], [.2, .8], [1, 3], ['k', 'r']):
+    ax_raw.plot(plt_times, idata[..., :200].T, color=color, alpha=alpha, lw=lw)
+ax_raw.set_title('Continuous training data')
+ax_raw.set_xlim([0, plt_times[-1]])
+ax_raw.legend([ax_raw.lines[0], ax_raw.lines[-1]], ['Inputs', 'Seed'])
+ax_raw.set_xlabel('Time (s)')
+ax_raw.set_ylabel('Amplitude (mV)')
+
+# Plot the predictions on the test set
+n_pred = int(4 * raw.info['sfreq'])
+time_test = np.arange(n_pred) / float(raw.info['sfreq'])
+ln_pred = ax_pred.plot(time_test, y_pred[:n_pred], color='r', alpha=.6, lw=2)
+# Plot true output signal
+ln_true = ax_pred.plot(time_test, y[ix_test][:n_pred],
+                       color='k', alpha=.3, lw=3)
+ax_pred.legend([ln_pred[0], ln_true[0]], ['Predicted', 'Actual'])
+ax_pred.set_title('Model predictions')
+ax_pred.set_xlabel('Time (s)')
+ax_pred.set_ylabel('Signal amplitude')
+ax_pred.set_xlim([0, time_test[-1]])
+plt.tight_layout()
+
+###############################################################################
+# Pull the coefficients for plotting
+coefs = get_coefs(get_final_est(pipeline))[0]
+coefs_delayed = coefs.reshape([-1, np.unique(delayer.delays_[:, 1]).shape[0]])
```

Comment:
(the reason that I'm returning this 2-D vector instead of just a 1-D vector of delay values is in order to support variable # delays for different features)